<launch>
     <node name="control_client" pkg="control_srv" type="client" respawn="false" output="screen" />
</launch>
